src/types/callable.js

```diff
@@ -62,14 +69,12 @@ export function parseCallableArgs(info) {
   };
 
   const initOutArgs = () => {
-    const outArgs = outArgsDetail.map((d) => initArgument(d.type));
-
-    return outArgs;
+    return new BigUint64Array(outArgsDetail.map((d) => initArgument(d.type)));
```

Comment:
This isn't quite correct. I think `initArgument` should return a pointer to a memory of the size of the interface (if given an interface) and not initialise a new object. Please advise!

Reply:
I'm not entirely sure about that. If you managed to do it the better way, introduce it in a new PR.